minecraft_servers/twerion/manifest.json

```diff
@@ -0,0 +1,95 @@
+{
+	"server_name": "twerion",
+	"nice_name": "Twerion Netzwerk",
+    "direct_ip": "twerion.net",
+    "server_wildcards": [
+      "%.twerion.net",
+	  "%.twerion.de",
+	  "%.twerion.com",
+	  "%.twerion.eu",
+	  "%.twerion.org"
+    ],
+    "supported_languages": [
+       "de",
+       "en",
+	   "ru"
+    ],
+    "social": {
+      "web": "https://twerion.net",
+      "web_shop": "https://shop.twerion.net",
+	  "web_support": "https://forum.twerion.net",
```

Comment:
Web Support can't be forum; add FAQ or contact form

Reply:
Removed.